<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addGames</class>
 <widget class="QMainWindow" name="addGames">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>261</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="labelLayout">
     <item>
      <widget class="QLabel" name="steamNameLabel">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="friendDescText">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="returnBackButton">
       <property name="text">
        <string>Вернуться назад</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addGamesToFriendButton">
       <property name="text">
        <string>Добавить игры другу</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>651</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="tablesLayout">
     <item>
      <widget class="QTableWidget" name="allGamesTable"/>
     </item>
     <item>
      <widget class="QTableWidget" name="pickedGamesTable"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
